.gitattributes

```diff
@@ -13,5 +13,9 @@
 *.properties text=auto eol=lf
 
 *.js text=auto eol=lf
+*.jsx text=auto eol=lf
+*.ts text=auto eol=lf
+*.tsx text=auto eol=lf
+*.json text=auto eol=lf
```

Comment:
thanks this helps

Reply:
In a future PR I'm thinking of blacklisting instead of whitelisting text files. I think the only non-text file in the repo are PNG and ICO.